charts/crunchy-postgres/values.yaml

```diff
@@ -113,18 +58,30 @@ patroni:
     pg_hba: "host all all 0.0.0.0/0 md5"
     parameters:
       shared_buffers: 16MB # default is 128MB; a good tuned default for shared_buffers is 25% of the memory allocated to the pod
-      wal_buffers: "-1" # automatically set as 1/32 of shared_buffers or 64kB, whichever is larger
+      wal_buffers: "64kB" # automatically set as 1/32 of shared_buffers or 64kB, whichever is larger
```

Comment:
is this in-line comment still valid after changing from -1 to 64kb?

Reply:
Nice catch, that was outdated so I slightly changed it so that information would still be there:

`this can be set to -1 to automatically set as 1/32 of shared_buffers or 64kB, whichever is larger`